src/dynamics/ads_player/ads_player.js

```diff
@@ -254,6 +255,9 @@ Scoped.define("module:Ads.Dynamics.Player", [
                         this.set("quartile", "fourth");
                     },
                     "ads:start": function(ev) {
+                        if (this.get('customclickthrough')) {
+                            this.get('customclickthrough').style.display = 'block';
```

Comment:
If the customer is supplying the custom click div, I feel like this would have to be handled more elegantly? The display may be something like `flex` or `table` instead? What do you think @jimee02 ? 

Maybe I can keep a reference to the original `display` style attribute?

Reply:
I would remove this completely if this was not currently an option.  What did the original customclickthrough element track?

Reply:
After talking to @Dawolee, we're keeping this capability